rebasehelper/plugins/build_log_hooks/files.py

```diff
@@ -191,6 +191,17 @@ def _get_best_matching_files_section(cls, rebase_spec_file, file):
 
         return best_match_section or rebase_spec_file.get_main_files_section()
 
+    @classmethod
+    def _sanitize_path(cls, path):
+        """Changes the path to follow Fedora Packaging Guidelines."""
+        if path.startswith('%{_mandir}'):
+            manpage_start = path.rfind('/')
```

Comment:
What's wrong with `os.path.basename()`? Or `os.path.split()`, as you need the ret of the path.